src/uwtools/drivers/ungrib.py

```diff
@@ -41,18 +42,27 @@ def __init__(
 
     # Workflow tasks
 
-    @task
-    def gribfile(self):
-        """
-        A symlink to the input GRIB file.
-        """
-        path = self._rundir / "GRIBFILE.AAA"
-        yield self._taskname(str(path))
-        yield asset(path, path.is_symlink)
-        infile = Path(self._driver_config["gfs_file"])
-        yield file(path=infile)
-        path.parent.mkdir(parents=True, exist_ok=True)
-        path.symlink_to(infile)
+    @tasks
+    def gribfiles(self):
+        """
+        Symlinks to all the GRIB files.
+        """
+        yield self._taskname("gribfiles")
+        gfs_files = self._driver_config["gfs_files"]
+        offset = abs(gfs_files["offset"])
+        endhour = gfs_files["forecast_length"] + offset + 1
```

Comment:
Is the +1 needed here for the for loop below to get an accurate range? Or is there an additional reason for the +1 to help get the correct endhour?

Reply:
This one is for the appropriate range.

Reply:
Could we move the `+ 1` to the `range()` call so that it's obvious what it's for, and so that `endhour` actually has the expected value? For example, if someone was in the debugger looking at `endhour`, it might be nice if it were not already incremented.